MAINTAINERS.md

```diff
@@ -32,20 +32,20 @@
   - [Using Milestones](#using-milestones)
 - [Mocha's Release Process](#mochas-release-process)
 - [Projects](#projects)
-- [About The JS Foundation](#about-the-js-foundation)
+- [About The OpenJS Foundation](#about-the-openjs-foundation)
 - [About OpenCollective](#about-opencollective)
 
 <!-- tocstop -->
 
 ## Introduction
 
-Hi stranger!  We've written this document for:
+Hi stranger! We've written this document for:
 
 1. Active maintainers of Mocha
 1. Prospective maintainers of Mocha
 1. Anyone curious about how Mocha's maintainers maintain Mocha
 
-The purpose of this document is to *describe our processes*.  We want to avoid conflicts and confusion around "unwritten rules".  In our opinion, the most straightforward way to address this is to *write them down*.  This *also* happens to be the most straightforward way to change them!
+The purpose of this document is to _describe our processes_. We want to avoid conflicts and confusion around "unwritten rules". In our opinion, the most straightforward way to address this is to _write them down_. This _also_ happens to be the most straightforward way to change them!
```

Comment:
Why have you changed all this? On my platform the result is the same for * and _.

Reply:
I believe our markdown-linter changed this.